### DISCUSSION TITLE
Suggestion to block 60fps videos in freetube.

### DISCUSSION THREAD

**Discussion by exefile2** [FreeTubeApp/FreeTube#6199] [General]
I’ve discovered an extension that can block 60fps videos, which is incredibly helpful for low-end PCs. Is there any way to integrate this functionality directly into FreeTube?
Im attaching the link below
https://chromewebstore.google.com/detail/h264ify/aleakchihdccplidncghkekgioiakgal
:)

**Comment by absidue** [comment]
Duplicate of #504
